src/Mutator/Removal/FunctionCallRemoval.php

```diff
@@ -28,6 +29,22 @@ protected function mutatesNode(Node $node): bool
             return false;
         }
 
-        return $node->expr instanceof Node\Expr\FuncCall;
+        if (!$node->expr instanceof Node\Expr\FuncCall) {
+            return false;
+        }
+
+        $name = $node->expr->name;
+
+        if (!$name instanceof Node\Name) {
+            return false;
```

Comment:
I think this can be `return true;`. Even if we don't know what function is called, because the name is a variable or a string for example, we can still remove it and see if it has an effect.

Reply:
Yep, that was a mistake, fixed and added a test :+1: